source/PlayerInfo.cpp

```diff
@@ -841,14 +841,30 @@ void PlayerInfo::AddShip(const shared_ptr<Ship> &ship)
 
 
 
-// Buy a ship of the given model, and give it the given name.
-void PlayerInfo::BuyShip(const Ship *model, const string &name)
+// Charge the player the cost of a ship and give it the specified name:
+void PlayerInfo::BuyShip(const Ship *model, const std::string &name)
```

Comment:
Why adding "std::" here (and on line 853)? std should be covered by an using statement at the top of the class.

Reply:
```suggestion
void PlayerInfo::BuyShip(const Ship *model, const string &name)
```
Not sure, will remove it.